mgmt/imgmgr/src/imgmgr.c

```diff
@@ -854,29 +854,38 @@ imgr_upload(struct mgmt_cbuf *cb)
 }
 
 void
-imgr_dfu_stopped(void)
+imgmgr_dfu_stopped(void)
 {
-    if (imgmgr_dfu_callbacks_fn->dfu_stopped_cb != NULL)
+    if (imgmgr_dfu_callbacks_fn && imgmgr_dfu_callbacks_fn->dfu_stopped_cb != NULL)
```

Comment:
this statements checks two pointers, one is compared to NULL other is treated as boolean value, both are correct but using two different styles in one line is strange.

Reply:
I do agree